public/jira/rollup/rollup.ts

```diff
@@ -8,9 +8,9 @@ import type { DerivedIssue } from "../derived/derive";
 
 import type { DerivedRelease } from "../releases/derive";
 
-export type IssueOrRelease<CustomFields = unknown> =
-  | (DerivedIssue & CustomFields)
-  | (DerivedRelease & CustomFields);
+export type IssueOrRelease<CustomFields = unknown> = (DerivedIssue &
```

Comment:
Are you sure you wanted to change this from `DerivedIssue | DerivedRelease` to `DerivedIssue & DerivedRelease`? Anything you assign to an `IssueOrRelease` will need all properties from _both_ `DerivedIssue` and `DerivedRelease` now.

Whichever you end up using, you can simplify this by pulling `CustomFields` out on its own: `(DerivedIssue | DerivedRelease) & CustomFields`.